exporter/sapmexporter/exporter_test.go

```diff
@@ -53,14 +53,6 @@ func TestCreateTracesExporter(t *testing.T) {
 	assert.NoError(t, te.Shutdown(context.Background()), "trace exporter shutdown failed")
 }
 
-func TestCreateTracesExporterWithInvalidConfig(t *testing.T) {
-	cfg := &Config{}
-	params := exportertest.NewNopCreateSettings()
-	te, err := newSAPMTracesExporter(cfg, params)
```

Comment:
Is the `Validate` with no endpoint covered somewhere else?

Reply:
yes, tested here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/eb8a8309909b4a909a307666eefab6657b282428/exporter/sapmexporter/config_test.go#L100